src/edit/canvas.js

```diff
@@ -0,0 +1,186 @@
+var gl;
```

Comment:
Oof -- are these required for the testing? I think we could push for `webgl-distort` to include testing rather than testing an external module. I wrote the upstream so that's on me... what do you think? I do agree it's good that it have tests.

Reply:
D'oh!  I meant to remove these (canvas and texture) - they're left from when I was playing with bringing in the extra files.

Reply:
aha do we still need to remove these? Could be almost there!